config.json

```diff
@@ -289,6 +289,20 @@
       ]
     },
     {
+      "uuid": "0cc485e9-43ba-4d97-a622-ee4cb8b9f1f7",
+      "slug": "crypto-square",
```

Comment:
this is, I believe, the only exercise that doesn't define `"core"` or `"unlocked_by"`

Interesting that configlet doesn't care. Does Nextercism care? Does this exercise show up on Nextercism, and if so, in what capacity?

Reply:
Far as I can tell it is a non-core already-unlocked (unlocked_by null) exercise. I did not read any code to verify this statement so I'm prepared to be wrong.

Reply:
It is interesting, but I'm not sure what to do with the information. I presume that if Nextercism cares, we'll discover it and fix it upon adoption. For now my basic attitude is that configlet is the current canonical judge of what is correct; as it doesn't care, the file is valid.